numpy/_core/tests/test_cpu_features.py

```diff
@@ -124,6 +124,7 @@ def load_flags_auxv(self):
         " therefore this test class cannot be properly executed."
     ),
 )
+@pytest.mark.thread_unsafe(reason="setup & tmp_path_factory threads-unsafe, modifies environment variables")
```

Comment:
Didn't we fix most of these issues? That said, the environment variable mutation isn't safe.